File: src/main/java/onl/tesseract/command/BuildToolsCommand.java
package onl.tesseract.command;

import onl.tesseract.CreativePlayer;
import onl.tesseract.commandBuilder.CommandContext;
import onl.tesseract.commandBuilder.annotation.Command;
import onl.tesseract.commandBuilder.annotation.CommandBody;
import onl.tesseract.menu.PluginsToolsMenu;
import onl.tesseract.menu.MenuMenu;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

@Command
public class BuildToolsCommand extends CommandContext {
    @CommandBody
    public boolean onCommand(CommandSender sender)
    {
        if (sender instanceof Player player )
        {
            new PluginsToolsMenu().open(player);
        }

        return true;
    }
}

File: src/main/java/onl/tesseract/command/BuildToolsCommand.kt
package onl.tesseract.command

import onl.tesseract.commandBuilder.CommandContext
import onl.tesseract.commandBuilder.annotation.Command
import onl.tesseract.commandBuilder.annotation.CommandBody
import onl.tesseract.menu.PluginsToolsMenu
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

@Command
class BuildToolsCommand : CommandContext() {
    @CommandBody
    fun onCommand(sender: CommandSender?): Boolean {
        if (sender is Player) {
            PluginsToolsMenu().open(sender)
        }

        return true
    }
}
